lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.cpp

```diff
@@ -122,7 +122,8 @@ void S1apStateConverter::enb_to_proto(enb_description_t* enb,
   proto->set_ran_cp_ipaddr_sz(enb->ran_cp_ipaddr_sz);
 
   // store ue_ids
-  hashtable_uint64_ts_to_proto(&enb->ue_id_coll, proto->mutable_ue_ids());
+  proto_map_uint32_uint64_to_proto(enb->ue_id_coll, proto->mutable_ue_ids());
```

Comment:
Can we just pass the enb->ue_id_coll reference to initialize the proto field? Eventually, this conversion should not be needed as the enb->ue_id_coll should be directly serialized to S1ap state.